src/main/resources/properties/configuration.properties

```diff
@@ -1,3 +1,3 @@
 configuration.profiles=
-configuration.suites=
+configuration.suites=web_app
 configuration.environments=
```

Comment:
[improvement suggestion] Would be great to restrict list of stories to run to exclude unnecessary stories. Since now allure report include "Star Wars API" story.